src/Controller/ItemsController.ts

```diff
@@ -92,4 +93,38 @@ export class ItemsController extends BaseHttpController {
 
     return this.ok()
   }
+
+  @httpDelete('/email-backups/:userUuid')
+  public async disableEmailBackups(request: Request): Promise<results.NotFoundResult | results.OkResult> {
+    const extensions = await this.itemRepository.findAll({
+      userUuid: request.params.userUuid,
+      contentType: ContentType.ServerExtension,
+      deleted: false,
+      sortBy: 'updated_at_timestamp',
+      sortOrder: 'DESC',
+    })
+
+    const deletedExtensions = extensions.filter(extension => {
+      if (!extension.content) {
+        return false
+      }
+
+      const decodedContent = this.contentDecoder.decode(extension.content)
+      if (!('subtype' in decodedContent) || decodedContent.subtype !== 'backup.email_archive') {
```

Comment:
Why negated operator here? Should go towards the affirmative direction:

```
return decodedContent.subtype! === 'backup.email_archive'
```

Reply:
See updates